src/components/Modal.tsx

```diff
@@ -117,6 +116,7 @@ class Modal extends React.Component<ModalProps, any> {
 				<ModalBackdrop z={8888} bg="rgba(0,0,0,0.4)" top right bottom left />
 				<ModalSizer style={props.containerStyle}>
 					<ModalPanel
+						p={['30px 20px', '30px 50px']}
```

Comment:
It would be great to use the sizings from the default theme ( https://github.com/balena-io-modules/rendition/blob/master/src/theme.ts#L97 ). 
`16px` all the way around looks quite good:

![screen shot 2019-01-14 at 09 17 35](https://user-images.githubusercontent.com/15064535/51104177-44250980-17dd-11e9-9b20-15ffdc13e949.png)

Reply:
@sradevski My bad, I mean to link to the `space` value https://github.com/balena-io-modules/rendition/blob/master/src/theme.ts#L87
You can use the `withTheme` helper from `styled-components` to pass the theme as a property to the component. Checkout the `Button` source for inspiration, its quite straightforward https://github.com/balena-io-modules/rendition/blob/master/src/components/Button.tsx#L245

Reply:
@LucianBuzzo oh yeah, the `space` variable. It's easy to do it, I was just wondering what approach would be more suitable for this PR. In that case, I will update the rest of the values of the Modal as well, if applicable.